### ISSUE TITLE
Component methods doesn't rise type errors.

### ISSUE THREAD

**Issue by krzysztof-grzybek** [sbrudz/ts-aframe-boilerplate#1]
Consider the following example:

```
AFRAME.registerComponent('hello-world', {
    init: function() {
       const a = this.multiply('customString');
   },

  multiply(myNumber: number) {
    return myNumber * 2;
  }
});
```

Line
`
const a = this.multiply('customString');
`
should throw error, but it doesn't.

Enabling strict mode should fix this problem. (precisely `noImplicitThis` with  [this PR](https://github.com/Microsoft/TypeScript/pull/14141)).

Would You accept PR?

**Comment by sbrudz** [comment]
This would be a very helpful check to have.  I'd be happy to accept a PR.  Thank you!

**Comment by sbrudz** [comment]
Resolved by #2
